uni/udb/breakpoint.icn

```diff
@@ -32,28 +32,27 @@ class BreakPoint(
 # Check the Monitored file for a breakpoint
 #
 method checkBreakpoint()
-   local  cur_file, cur_line, L, bp, id
-
+   local  cur_file, cur_line, L, bp, id, bpTable
    cur_file := keyword("file", MONITORED)
    if member(breakPoints, cur_file) then {
       cur_line := keyword("line", MONITORED)
       L := breakPoints[cur_file]
-      every bp := !L do
-	 if cur_line = bp.line & bp.state = ENABLED &
-	    id := isBreakExist(cur_file,cur_line) &
+      every bp := !L do {
+	      if cur_line = bp.line & bp.state = ENABLED &
```

Comment:
indentation

Reply:
I believe this is a GitHub issue, actual code looks like this.
![Screenshot 2024-01-04 160108](https://github.com/uniconproject/unicon/assets/77177242/df22d7c0-6e4a-49a0-92da-bc2f7ab8ad10)

Reply:
OK. Just make sure you don't have a mix type of whitespace, tabs vs space.

Reply:
Looks like that was the issue, one tab and 6 spaces.